### PR TITLE
Placing blocks doc

### PR DESCRIPTION
**Pull request by dannylamb**
**GitHub Issue**: #1040 

# What does this Pull Request do?

Documentation for how to place blocks with both core block placement and context.

# What's new?

- A markdown file
- A bunch o' screenshots

# How should this be tested?

Read it and try it.

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: docs/user-documentation/placing-blocks.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+![Main content region](../assets/main_content_region.png)
+
+If you wanted to move the content above its tabs, you could drag the "Main page content" block above the "Tabs" block.
+If you scroll all the way down to the bottom and click the "Save blocks" button, now when you go to view content you
+should see something like this
+
+![Content above tabs](../assets/content_above_tabs.png)
+
+If you want to add a new block to a region, hit the "Add block" button for that region and you'll be brought to a modal
+that will let you select which block to place.
```
Comment (mjordan) [inline docs/user-documentation/placing-blocks.md]: The button says "Place block", not "Add block".

File: docs/user-documentation/placing-blocks.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# Placing Blocks
+
```
Comment (mjordan) [inline docs/user-documentation/placing-blocks.md]: Suggest adding a secondary (2-hash) heading here "Using Block Layout".

File: docs/user-documentation/placing-blocks.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+whether you are showing or hiding the block for this list of pages.  We want to show this only on the front page, so we'll
+leave it set at "Show for the listed pages".  Click "Save block" and go back to a piece of content.  You shouldn't see
+the development menu.
+
+![No devel menu](../assets/no_devel_menu.png)
+
+But go to the frontpage and...
+
+![Still got that ugly devel menu](../assets/still_got_that_ugly_devel_menu.png)
+
```
Comment (mjordan) [inline docs/user-documentation/placing-blocks.md]: Suggest adding a secondary (2-hash) heading here "Using Context".

**Comment by mjordan** [comment]
I approved this PR but requested a couple of minor changes. Could someone take a look at these suggestions, and if you're OK with them, add them, and I'll merge? I'd do it myself but can't because I'm reviewing.

**Comment by dannylamb** [comment]
Thanks for the feedback @mjordan.  I've pushed up some changes to do what you've asked.  If you're happy, merge away.

**Pull request merged**